adafruit_debouncer.py

```diff
@@ -90,19 +111,29 @@ def _get_state(self, bits):
 
     def update(self):
         """Update the debouncer state. MUST be called frequently"""
-        now = time.monotonic()
+        now = MONOTONIC_TIME()
         self._unset_state(_CHANGED_STATE)
         current_state = self.function()
         if current_state != self._get_state(_UNSTABLE_STATE):
             self.previous_time = now
             self._toggle_state(_UNSTABLE_STATE)
         else:
-            if now - self.previous_time >= self.interval:
+            if now - self.previous_time >= self._interval:
                 if current_state != self._get_state(_DEBOUNCED_STATE):
                     self.previous_time = now
                     self._toggle_state(_DEBOUNCED_STATE)
                     self._set_state(_CHANGED_STATE)
 
+    @property
+    def interval(self):
+        """The debounce delay, in seconds"""
+        return self._interval / MONOTONIC_UNITS_PER_SEC
+
+
+    @interval.setter
+    def interval(self, new_interval_s):
+        self._interval = new_interval_s * MONOTONIC_UNITS_PER_SEC
```

Comment:
Pylint is failing on the following:
```
************* Module adafruit_debouncer
C: 27, 0: Line too long (104/100) (line-too-long)
W:135, 8: Attribute '_interval' defined outside __init__ (attribute-defined-outside-init)
```
It's possible that simply adding `self._interval = None` into `init` could solve it, but I'm not certain how that would affect the rest of the code, so I don't want to make that change myself without better understanding the rest of what's going on here.

Reply:
OK, it's fine with me to explicitly set the underlying attribute in `__init__`, done :)